packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -36,11 +44,11 @@ export type BannerProps = {
    * ("status" for something that dynamically updates, "alert" for errors, null for something
    * unchanging)
    */
-  role?: "status" | "alert";
+  role?: (typeof bannerRoleValues)[number];
   /**
    * Determine the color and icon of the alert
    */
-  severity: AlertColor;
+  severity: (typeof bannerSeverityValues)[number];
```

Comment:
Do we still want this based on `AlertColor`? I'll put an example in another comment.